MAINTAINERS.md

```diff
@@ -0,0 +1,274 @@
+
+* Can customers understand at a high level what changed in this release?
+* Is there a link to the documentation where they can read more about each main change?
+* Are there any graphics or code snippets that can enhance readability?
+* Are we calling out any key contributor(s) to this release?
+  - All contributors are automatically credited, use this as an exceptional case to feature them
+
+Once you're happy, hit `Publish release`. This will kick off the [Publishing workflow](https://github.com/awslabs/aws-lambda-powertools-python/actions/workflows/publish.yml) and within a few minutes you should see the latest version in PyPi, and all issues labeled as `pending-release` will be notified.
+
+> TODO: Wait for @am29d new Lambda Layers pipeline work to complete, then add how Lambda Layers are published
+
+### Releasing a documentation hotfix
+
+You can rebuild the latest documentation without a full release via this [GitHub Actions Workflow](https://github.com/awslabs/aws-lambda-powertools-python/actions/workflows/rebuild_latest_docs.yml). Choose `Run workflow`, keep `develop` as the branch, and input the latest Powertools version available.
+
+This workflow will update both user guide and API documentation.
+
+### Maintain Overall Health of the Repo
+
+> TODO: Coordinate removing `master` and renaming `develop` to `main`
+
+Keep the `develop` branch at production quality at all times. Backport features as needed. Cut release branches and tags to enable future patches.
+
+### Manage Roadmap
+
+See [Roadmap section](https://awslabs.github.io/aws-lambda-powertools-python/latest/roadmap/)
+
+Ensure the repo highlights features that should be elevated to the project roadmap. Be clear about the feature’s status, priority, target version, and whether or not it should be elevated to the roadmap.
+
+### Add Continuous Integration Checks
+
+Add integration checks that validate pull requests and pushes to ease the burden on Pull Request reviewers. Continuously revisit areas of improvement to reduce operational burden in all parties involved.
+
+### Negative Impact on the Project
+
+Actions that negatively impact the project will be handled by the admins, in coordination with other maintainers, in balance with the urgency of the issue. Examples would be [Code of Conduct](CODE_OF_CONDUCT.md) violations, deliberate harmful or malicious actions, spam, monopolization, and security risks.
+
+### Becoming a maintainer
+
+In 2023, we will revisit this. We need to improve our understanding of how other projects are doing, their mechanisms to promote key contributors, and how they interact daily.
+
+We suspect this process might look similar to the [OpenSearch project](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md#becoming-a-maintainer).
+## Common scenarios
```

Comment:
Maybe we could add additional paragraph  to mantainers responsibility to constantly evaluate repeating cases and update common scenarios if they feel all group would benefit from it. I think it make sense since we want this section to evolve over time as we grow and we experience more :-)

Reply:
Great idea too. Added them at the top:

> Be aware of recurring ambiguous situations and [document them](#common-scenarios) to help your fellow maintainers.

Added this paragraph within Common scenarios section to reemphasize their purpose and defer to maintainer's judgement.

> These are recurring ambiguous situations that new and existing maintainers may encounter. They serve as guidance. It is up to each maintainer to follow, adjust, or handle in a different manner as long as [our conduct is consistent](#uphold-code-of-conduct)

Let me know if it suffices.